quiz.html

```diff
@@ -39,61 +39,61 @@ <h1>Skin Type Quiz</h1>
 				<!-- This form is for the skin type quiz and is different to the other pages -->
 				<form name="quiz" id="quiz">
 					<div>
-						<h1>How does your skin typically feel throughout the day?</h1>
+						<h1>How does your skin typically feel in the morning?</h1>
 						<h2>1 / 5</h2>
 						<div class="choices">
-							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Oily">Oily</p></div>
-							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Dry">Dry</p></div>
-							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Normal">Normal</p></div>
-							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question1" value="Sensitive">Sensitive</p></div>
+							<div class="choicebox"><img src="images/oily.png"><p><input type="radio" name="question1" value="Oily">Oily</p></div>

```

Comment:
Included icons that match each option for the first question in my skin type quiz.